README.md

```diff
@@ -36,15 +36,27 @@ Der Server bietet sowohl für die App als auch für den Bot die notwendigen API
 Der Source-Code und Dokumentation des Servers ist in diesem Repo zu finden.
 
 #### Install and Start
-Voraussetzung ist ein installiertes python 3.8 package
-- `sudo ./manage.sh install` um den server zu installieren
-  (sudo only necessary to install graphviz to display story graph)
-- `./manage.sh start` um den server zu starten
-- für weitere befehle `./manage.sh help` benutzen
+Voraussetzung ist ein Linux System mit installiertem python 3.8
+1. `./manage.sh install` um den server zu installieren
+1. `./manage.sh start` um den server zu starten
+1. für weitere befehle `./manage.sh help` benutzen
+1. siehe Bot repo für Installation und Start des Bots
+1. nach Start von Server und Bot kann die App und damit das Spiel gestartet werden
 
 #### Architektur-Überblick
 Der Server wird durch eine Flask App (`/app`) implementiert. Diese verwaltet verschiedene API-Endpunkte (`/app/routes`). Außerdem wird eine Datenbank verwaltet. Deren ORM wird in `/app/models` implementiert. Der Story-Inhalt und der Story-verwaltende Code liegt unter `/app/story`.  
 
+#### Nutzung ohne App
+Unsere App ist sehr datenhungrig. Um das Spiel zu testen, ohne die App zu installieren, haben wir ein Postman package erstellt, welches die App immitiert mit dem sich das Spiel durchspielen lässt (die Spielerfahrung leidet darunter jedoch deutlich). Die Nutzung ist wie folgt:
+1. `detective-game-no-app-walkthrough.json` mit Postman importieren
+1. `create user` senden, um einen neuen Nutzer zu erstellen
+1. die URL aus der response öffnen, um den Chat mit Kommissar Rex zu starten
+1. `send mocked contacts` senden, um das Stehlen von Kontakten für "Personalisierung" zu mocken
+1. Die Story spielen und Tasks wie folgt erfüllen:
+   - die request für den entsprechenden task senden
```

Comment:
This does not work for me on an old version (the chrome one) of Postman as the pre-request script fails as `pm` is not defined.
I think that this should work with the postman desktop application. @robinwersich pls test this

We should add a note here that the desktop version of postman might be needed for this to work

Reply:
i tested this on the desktop version, it works -  will add a note